static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -559,7 +559,7 @@ function WidgetBuilder({
     const aggregateAliasFieldStrings =
```

Comment:
https://github.com/getsentry/sentry/pull/34181/commits/6cafad6245d2a5b8ce222e8128186a12343ec012

Good point, I've renamed them to `fieldStrings` and `prevAggregateFieldStrings`. I also went ahead and did the consolidation you suggested, but instead of doing the ternary to choose the ternary like you suggested, I just kept the callback like:
```js
.map(field => dataSet === DataSet.RELEASES ? stripDerivedMetricsPrefix(field) : field)
```

I wanted to use lodash's identity function that does `value => value` but it seems to change the resulting type to `(string | undefined)[]` (and caused some ts errors) so I decided not to do that and keep it as the above

Reply:
👍  nice, looks good to me!